apps/api/tests/test_user_apply.py

```diff
@@ -260,3 +275,19 @@ def test_application_data_is_bson_encodable() -> None:
     data.linkedin = HttpUrl("https://linkedin.com")
     encoded = bson.encode(EXPECTED_APPLICATION_DATA.model_dump())
     assert len(encoded) == 415
+
+
+@patch("utils.email_handler.send_application_confirmation_email", autospec=True)
+@patch("services.mongodb_handler.update_one", autospec=True)
+@patch("routers.user.datetime", autospec=True)
+@patch("services.gdrive_handler.upload_file", autospec=True)
+@patch("services.mongodb_handler.retrieve_one", autospec=True)
```

Comment:
Chore: pretty sure we don't need all of these patches. I think all we care about is retrieve_one is not called. See `test_apply_with_invalid_data_causes_422`